From: Fabio Estevam <fabio.estevam@nxp.com>
Date: Fri, 8 Jan 2016 10:19:16 -0200
Subject: net: lpc_eth: Remove unused variables
Patch-mainline: v4.5-rc1
Git-commit: 541b8e291f29e480fb9c6400ce796c2eaefe9229
References: bsc#1026030 FATE#321670

Commit e7f4dc3536a400 ("mdio: Move allocation of interrupts into core")
introduced the following build warnings:

drivers/net/ethernet/nxp/lpc_eth.c: In function 'lpc_mii_init':
drivers/net/ethernet/nxp/lpc_eth.c:865:1: warning: label 'err_out_1' defined but not used [-Wunused-label]
drivers/net/ethernet/nxp/lpc_eth.c:826:20: warning: unused variable 'i' [-Wunused-variable]

Remove the unused variables to fix them.

Reported-by: Olof's autobuilder <build@lixom.net>
Signed-off-by: Fabio Estevam <fabio.estevam@nxp.com>
Acked-by: Andrew Lunn <andrew@lunn.ch>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/nxp/lpc_eth.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/net/ethernet/nxp/lpc_eth.c
+++ b/drivers/net/ethernet/nxp/lpc_eth.c
@@ -823,7 +823,7 @@ static int lpc_mii_probe(struct net_devi
 
 static int lpc_mii_init(struct netdata_local *pldat)
 {
-	int err = -ENXIO, i;
+	int err = -ENXIO;
 
 	pldat->mii_bus = mdiobus_alloc();
 	if (!pldat->mii_bus) {
@@ -862,7 +862,6 @@ static int lpc_mii_init(struct netdata_l
 
 err_out_unregister_bus:
 	mdiobus_unregister(pldat->mii_bus);
-err_out_1:
 	mdiobus_free(pldat->mii_bus);
 err_out:
 	return err;
